src/components/NewsletterView.js

```diff
@@ -23,17 +30,35 @@ class NewsletterView extends React.Component {
     if (!issue) {
       return null;
     }
+    this.props.history.listen(stuff =>
+      this.issueRef.current ? this.issueRef.current.focus() : null
+    );
+
+    const date = new Date('01/' + issue.Date); // Add 01/ to be valid date, e.g. 01/01/2019
```

Comment:
I would go for something like this when constructed the formatted date:

```javascript
const (month, year) = issue.Date.split('/')
const issueDate = new Date(year, month - 1)
const dateFormatted = new Intl.DateTimeFormat('en-US', {
    month: 'short',
    year: 'numeric'
}).format(issueDate)
```

Long run we should consider having a processing step that massages the TOC yaml into a format so that we don't have to do string parsing and `issues.Date` is already a Date object instead of the raw values when we access it for the NewsletterView

If you want to stick with the current solution I would suggest adding to the comment to indicate that the `01/` you're adding is the _day_ and not the month.

Reply:
Seems like a nicer and more readable (and also probably more correct) way to do it. I like the idea of processing the yaml too, hadn't though of that.